docs/Configuration-Properties.md

```diff
@@ -28,9 +28,9 @@ The PowerAuth Server uses the following public configuration properties:
 
 | Property                                                      | Default   | Note                                                                                    |
 |---------------------------------------------------------------|-----------|-----------------------------------------------------------------------------------------|
-| `powerauth.service.crypto.activationValidityInMilliseconds`   | `120000`  | Default activation validity period in miliseconds                                       |
+| `powerauth.service.crypto.activationValidityTime`   | `PT2M`  | Default activation validity period (ISO 8601 Duration format) |
```

Comment:
May we use the variant without `PT` prefix? Spring Boot can do that, see https://docs.spring.io/spring-boot/docs/2.1.13.RELEASE/reference/html/boot-features-external-config.html#boot-features-external-config-conversion-duration

Reply:
While it is true that we can omit the 'PT' prefix and still have Spring Boot interpret the duration correctly, I believe we should consider the implications for broader public documentation and usage. By adhering to the ISO 8601 Duration format in its complete form, including the 'PT' prefix, we ensure that our format is standard-compliant and immediately recognizable to anyone familiar with ISO 8601.

Reply:
Standards are fine. Let's wait for another opinion and we will decide.